examples/demo_base.rb

```diff
@@ -1,6 +1,6 @@
 require 'pp'
 
-$:.unshift 'lib'
+$:.unshift '../lib'
```

Comment:
Let's make the proper `File.expand_path('../lib', __FILE__)`, then :) 
Because I run them from root of repo (`ruby examples/<something>.rb`), and this way they will not work.